multiaddr_test.go

```diff
@@ -166,6 +166,7 @@ func TestBytesToString(t *testing.T) {
 	testString("/ip4/127.0.0.1/udp/1234", "047f0000011104d2")
 	testString("/ip4/127.0.0.1/tcp/4321", "047f0000010610e1")
 	testString("/ip4/127.0.0.1/udp/1234/ip4/127.0.0.1/tcp/4321", "047f0000011104d2047f0000010610e1")
+	testString("/onion/aaimaq4ygg2iegci:80", "bc030010c0439831b48218480050")
```

Comment:
I would say separating out the port is better too, but I followed what the existing partial implementation was already doing. My understanding is that it is tcp.